README.adoc

```diff
@@ -9,61 +9,70 @@ endif::[]
 
 NOTE: This template is in progress. It should be working, but is intended to have a lot more features.
 
+The template includes a server with a mock database and https://en.wikipedia.org/wiki/Cross-site_request_forgery[CSFR] protection and a client with login. It also integrates https://github.com/nubank/workspaces/[Nubank Workspaces] for a visual, interactive development environment for Fulcro components, which you can choose to use. Both backend Clojure and frontend ClojureScript REPLs are configured and started. There are also sample frontend tests.
+
+The server code leverages https://github.com/tolitius/mount[Mount] to make it easy to start and to reload changes - see the helper functions in link:https://github.com/fulcrologic/fulcro-template/blob/master/src/dev/user.clj[`src/dev/user.clj`]. Client hot code reloading is handled automatically by https://shadow-cljs.org/[shadow-cljs].
+
+The main project source is in `src/main`, the frontend code in `.cljs` files in `+app.*+` and `+app.ui.*+`.
+
 Dependency Aliases:
 
 You will want to enable the `:dev` dependency while developing this project.  In IntelliJ this is in the
 "Clojure Deps" border tab window under "Aliases".
 
-The main project source is in `src/main`.
-
 == Quick Start
 This is a manual to get started with this example project.
 It comes with `npm run` scripts pre-loaded, you can find them in the package.json file.
 
+Typically you want to start the server and the _main_ and/or _workspaces_ (if you decided to use Nubank Workspaces) client builds. See below. You can simply start everything from the command line
+
 ```Shell
 git clone git@github.com:fulcrologic/fulcro-template.git fulcro-app
 cd fulcro-app
 
 # The shadow-cljs compiler is a dependency.
-yarn install
-# Of course you can use npm if you like.
-
-#### Develop components with cljs ####
-# Workspaces:
-
-npm run client/workspaces
-
-# Visit http://localhost:8023
-# Have a look at src/workspaces and https://github.com/nubank/workspaces
+yarn install # or: npm install
 
-#### Refreshing tests in the browser ####
-# CLJS Tests:
-npm run client/test
-# Visit http://localhost:8022
-
-#### full-stack development ####
-# Start the cljs compiler for the main target
-npm run client/main
```

Comment:
I'm personally not a fan of npm run...this was someone else's suggestion.  I'd much rather ppl just see the raw command.

Reply:
I see your point. Changed.